tests/nnstreamer_repo/runTest.sh

```diff
@@ -32,16 +32,16 @@ fi
 convertBMP2PNG
 
 # Fail Test : Negotiation Error (dimension)
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} multifilesrc location=testsequence_%1d.png index=0 caps=\"image/png,framerate=(fraction)30/1\" ! pngdec ! tensor_converter ! queue ! tensor_reposink silent=false slot-index=0 tensor_reposrc silent=false slot-index=0 caps=\"other/tensor,dimension=(string)3:100:100:1,type=(string)uint8,framerate=(fraction)30/1\" ! multifilesink location=testsequence01_%1d.log" 0F_n 0 1 $PERFORMANCE
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} multifilesrc location=testsequence_%1d.png index=0 caps=\"image/png,framerate=(fraction)3/1\" ! pngdec ! tensor_converter ! queue ! tensor_reposink silent=false slot-index=0 tensor_reposrc silent=false slot-index=0 caps=\"other/tensor,dimension=(string)3:100:100:1,type=(string)uint8,framerate=(fraction)3/1\" ! multifilesink location=testsequence01_%1d.log" 0F_n 0 1 $PERFORMANCE
```

Comment:
What if we simple do NOT specify framerate?

Reply:
Actually, the best way is to set ```framerate=(fraction)0/1```. However, we have some incompatible elements :( (don't know if it's because of tensor-repo or not)

Reply:
If the frame rate set to 0, the pipeline does not work..
And the frame rate cannot be omitted in multifilesrc
```
#GstMultiFileSrc:caps property or a capsfilter to force to caps containing a framerate. Otherwise image decoders send EOS after the first picture.
```
It also failed when the frame rate was ignored only in tensor_reposrc.